### DISCUSSION TITLE
JSON.parse(val) in Datenpunkt Stil

### DISCUSSION THREAD

**Discussion by hosabeni** [Zefau/ioBroker.jarvis#2562] [Q&A]
Sollte `{"JSON.parse(val).Battery < 30":{"color":"LightCoral"}}` im Datenpunkt Stil funktionieren oder muss ich das anders machen?
Version: v3.2.0-alpha.49

![grafik](https://github.com/Zefau/ioBroker.jarvis/assets/77939385/5496199c-cd02-41ff-957b-b1ee14b282b9)

**Comment by mcuiobroker** [comment]
scheint nicht mehr zu funktionieren.
Auch bei Anzeigevariante geht es nicht mehr.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/cc46431d-1244-4fff-9419-e1d4ce3757d4)

![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/02b148d3-6f38-4cb1-b22e-1a2a4f1e4b78)
```
{"JSON.parse({val}).battery<30":{"color":"LightCoral"}}

{"default":"JSON.parse({val}).battery"}
```
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/188005a1-9dd3-48a0-a7d9-b48ddcc8d2ca)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/178037b5-5bbc-40b8-91df-04b880be1824)

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/2563
